docs/development/design/network-router.md

```diff
@@ -0,0 +1,93 @@
+# Network Router
+
+> Additional details and running code in [PR #981](https://github.com/nexodus-io/nexodus/pull/981)
+
+## Summary
+
+Add the ability for device's running nexd to access non-nexd devices and act as a router for external networks.
+
+## Proposal
+
+There are two supported use cases.
+
+1. Traffic from the Nexd mesh can reach external non-nexd nodes via source NAT without any configuration required.
+2. Traffic from the Nexd mesh can reach external non-nexd nodes but require a configuration for return traffic since SNAT is disabled. This scenario will also allow external devices to initiate traffic to the mesh since the devices are required to have a route to an interface on the device connected to the physical network. With SNAT being disabled, the non-nexd device can also initiate traffic from outside the mesh going in.
+
+### Network Router with SNAT
+
+This is a zero touch provisioning for the non-nexodus device since the edge nexd device will be source natting the traffic.
+
+```shell
+nexd --username admin --password floofykittens router --child-prefix 10.180.0.0/20 --network-router  https://try.nexodus.127.0.0.1.nip.io
+```
+
+Here is an example with text diagramming a packet flow.
+
+![no-alt-text](diagrams/network-router-snat.png)
+
```

Comment:
Maybe we should show that route table entries for 192.168.0.0 network are created by nexd in Node2